pcsx2/GameDatabase.cpp

```diff
@@ -14,63 +14,193 @@
  */
 
 #include "PrecompiledHeader.h"
+
 #include "GameDatabase.h"
 
-BaseGameDatabaseImpl::BaseGameDatabaseImpl()
-	: gHash( 9900 )
-	, m_baseKey( L"Serial" )
+#include "fmt/core.h"
+#include "yaml-cpp/yaml.h"
+
+std::string GameDatabaseSchema::GameEntry::memcardFiltersAsString()
 {
+	if (memcardFilters.empty())
+		return "";
+
+	std::string filters;
+	for (int i = 0; i < memcardFilters.size(); i++)
```

Comment:
Size is unsigned, so to prevent a warning, I'd do:
for (u32 i = 0; i < memcardFilters.size(); i++)